TROUBLESHOOTING.md

```diff
@@ -41,45 +41,45 @@ To read the error message returned by SendGrid's API:
 
 ```go
 func main() {
-	from := mail.NewEmail("Example User", "test@example.com")
-	subject := "Hello World from the SendGrid Go Library"
-	to := mail.NewEmail("Example User", "test@example.com")
-	content := mail.NewContent("text/plain", "some text here")
-	m := mail.NewV3MailInit(from, subject, to, content)
-
-	request := sendgrid.GetRequest(os.Getenv("SENDGRID_API_KE"), "/v3/mail/send", "https://api.sendgrid.com")
-	request.Method = "POST"
-	request.Body = mail.GetRequestBody(m)
-	response, err := sendgrid.API(request)
-	if err != nil {
-		log.Println(err)
-	} else {
-		fmt.Println(response.StatusCode)
-		fmt.Println(response.Body)
-		fmt.Println(response.Headers)
-	}
+    from := mail.NewEmail("Example User", "test@example.com")
+    subject := "Hello World from the SendGrid Go Library"
+    to := mail.NewEmail("Example User", "test@example.com")
+    content := mail.NewContent("text/plain", "some text here")
+    m := mail.NewV3MailInit(from, subject, to, content)
+
+    request := sendgrid.GetRequest(os.Getenv("SENDGRID_API_KE"), "/v3/mail/send", "https://api.sendgrid.com")
+    request.Method = "POST"
+    request.Body = mail.GetRequestBody(m)
+    response, err := sendgrid.API(request)
+    if err != nil {
+        log.Println(err)
+    } else {
+        fmt.Println(response.StatusCode)
+        fmt.Println(response.Body)
+        fmt.Println(response.Headers)
+    }
 }
 ```
 
-__CAUTION__: A non-2xx status code doesn't cause an error on sendgrid.API and the application has to verify the response:
+__CAUTION__: A non-2xx status code doesn't cause an error on sendgrid.API and the application have to verify the response:
```

Comment:
I think the original is more common usage.

Reply:
Changed Accordingly